recipes/libcurl/all/conanfile.py

```diff
@@ -207,13 +204,22 @@ def build_requirements(self):
         else:
             self.build_requires("libtool/2.4.6")
             self.build_requires("pkgconf/1.7.4")
-            if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            if self._settings_build.os == "Windows" and not get_env("CONAN_BASH_PATH"):
```

Comment:
```suggestion
            if self._settings_build.os == "Windows" and not get_env("CONAN_BASH_PATH") and self.conf.get("tools.microsoft.bash:path"):
```
Those env vars will be deprecated in the future.
https://docs.conan.io/en/latest/systems_cross_building/windows_subsystems.html#running-commands-inside-the-subsystem